lib/resty/shdict/simple.lua

```diff
@@ -54,7 +55,14 @@ function _M.gen_shdict_methods (opts)
                 end
 
                 -- with force:
-                ok, err = shdict:set(key, value, ttl)
+                local retries = 0
```

Comment:
This variable name looks confusing. We are actually recording the number of "tries" in this variable, not "retries".